sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3266,16 +3253,62 @@ SIREPO.app.directive('jobsList', function(requestSender, appState, $location, $s
                 );
             }
 
+            $scope.endSimulation = function(rowIndex) {
+                let s = $scope.data.rows[rowIndex][1];
+                let r = $scope.data.rows[rowIndex][7];
```

Comment:
Temporary variables like this aren't needed. They create places where state gets spread around which tend to cause errors and increase the size of methods. In addition, the `request` key inside of the j object isn't needed below we just say j.request. This could be condensed to 

```js
const r = {
  simulationId: $scope.data.rows[rowIndex][1], 
  report: $scope.data.rows[rowIndex][7], 
  simulationType: $scope.data.rows[rowIndex][0]
}
```